pkgs/development/compilers/unison/default.nix

```diff
@@ -31,14 +33,18 @@ stdenv.mkDerivation (finalAttrs: {
 
   nativeBuildInputs = [ makeWrapper ]
     ++ lib.optional (!stdenv.isDarwin) autoPatchelfHook;
-  buildInputs = lib.optionals (!stdenv.isDarwin) [ ncurses6 zlib gmp ];
+  buildInputs = lib.optionals (!stdenv.isDarwin) [ gmp ncurses6 zlib ];
 
   installPhase = ''
     mkdir -p $out/bin
-    mv ucm $out/bin
+    mv runtime $out/runtime
     mv ui $out/ui
-    wrapProgram $out/bin/ucm \
+    mv unison $out/unison
+    makeWrapper $out/unison/unison $out/bin/ucm \
+      --prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ libb2 ]} \
+      --prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ openssl ]} \
```

Comment:
```suggestion
      --prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ libb2 openssl ]} \
```

Reply:
Oh thanks! I had pending to check if this worked as expected, and it does :)